script.js

```diff
@@ -1,17 +1,49 @@
-$('.save-btn').on('click', createIdea);
+$('.save-btn').on('click', createCard);
 $('.new-idea-section').on('click', deleteIdea);
```

Comment:
You are assigning multiple event handlers to do different things, which can have side effects. 
Each event handler is given the chance to handle the event, however, if one of the handlers stops the event from propagating it might cause your code to break since they are all listening for the same event.

You have a class on your delete button so select it directly.
`$('.delete-btn').on('click', deleteIdea);`
you can do the same for the upvote and downvote buttons